.github/CODEOWNERS

```diff
@@ -7,4 +7,4 @@
 /eng/common/                                   @dotnet-maestro-bot @dougbu
 /eng/Versions.props                            @dotnet-maestro-bot @dougbu
 /eng/Version.Details.xml                       @dotnet-maestro-bot @dougbu
-/src/Razor/                                    @NTaylorMullen @ryanbrandenburg @TanayParikh @allisonchou
+/src/Razor/                                    @NTaylorMullen @ryanbrandenburg @TanayParikh @allisonchou @sharwell
```

Comment:
💡 The right hand side should be a group, not individuals

Reply:
Done! Lets see how this goes 😄